### ISSUE TITLE
Duke's Stories page needs h1

### ISSUE THREAD

**Issue by caitlynmayers** [caitlynmayers/dukes#142]
### Location
http://test-dukes.pantheonsite.io/stories/

### Description
The stories page doesn't have an h1.

**Comment by caitlynmayers** [comment]
Fix verified.
